README.md

```diff
@@ -382,151 +424,454 @@ Jest uses Jasmine 2 by default. An introduction to Jasmine 2 can be found
+```js
+describe('adding numbers', () => {
+  it('works sanely with simple decimals', () => {
+    expect(0.2 + 0.1).toBeCloseTo(0.3, 5);
+  });
+});
+```
 
-In these rare scenarios you can use this API to manually fill the slot in the module system's mock-module registry.
+The default for `numDigits` is 2, which has proved to be a good default in most cases.
 
-*Note It is recommended to use [`jest.mock()`](https://facebook.github.io/jest/docs/api.html#jest-mock-modulename-factory) instead. The `jest.mock` API's second argument is a module factory instead of the expected exported module object.*
+### `.toBeDefined()`
 
-### `jest.unmock(moduleName)`
-Indicates that the module system should never return a mocked version of the specified module from `require()` (e.g. that it should always return the real module).
+Use `.toBeDefined` to check that a variable is not undefined. For example, if you just want to check that a function `fetchNewFlavorIdea()` returns *something*, you can write:
 
-The most common use of this API is for specifying the module a given test intends to be testing (and thus doesn't want automatically mocked).
+```js
+describe('fetching new flavor ideas', () => {
+  it('returns something', () => {
+    expect(fetchNewFlavorIdea()).toBeDefined();
+  });
+});
+```
 
-*Note: this method was previously called `dontMock`. When using `babel-jest`, calls to `unmock` will automatically be hoisted to the top of the code block. Use `dontMock` if you want to explicitly avoid this behavior.*
+You could write `expect(fetchNewFlavorIdea()).not.toBe(undefined)`, but it's better practice to avoid referring to `undefined` directly in your code.
 
-### `jest.useFakeTimers()`
-Instructs Jest to use fake versions of the standard timer functions (`setTimeout`, `setInterval`, `clearTimeout`, `clearInterval`, `nextTick`, `setImmediate` and `clearImmediate`). Fake timers are used by default.
+### `.toBeFalsy()`
 
-### `jest.useRealTimers()`
-Instructs Jest to use the real versions of the standard timer functions.
+Use `.toBeFalsy` when you don't care what a value is, you just want to ensure a value is false in a boolean context. For example, let's say you have some application code that looks like:
+
+```js
+drinkSomeLaCroix();
+if (!getErrors()) {
+  drinkMoreLaCroix();
+}
+```
+
+You may not care what `getErrors` returns, specifically - it might return `false`, `null`, or `0`, and your code would still work. So if you want to test there are no errors after drinking some La Croix, you could write:
+
+```js
+describe('drinking some La Croix', () => {
+  it('does not lead to errors', () => {
+    drinkSomeLaCroix();
+    expect(getErrors()).toBeFalsy();
+  });
+});
+```
+
+In JavaScript, there are six falsy values: `false`, `0`, `''`, `null`, `undefined`, and `NaN`. Everything else is truthy.
```

Comment:
Oh wait no. This is as intended. It's just supposed to be a short note that explains "truthy" vs "falsy" & I think this works for that.